.github/workflows/ci.yml

```diff
@@ -32,3 +32,5 @@ jobs:
     steps:
     - uses: actions/checkout@master
     - uses: gaurav-nelson/github-action-markdown-link-check@v1
+      with:
+          config-file: 'markdown-link-check.json'
```

Comment:
Nit: given that this is part of the CI configuration specific to github, I would prefer to move it into the `.github` folder. Of course, provided it still does what it is supposed to do! :)